### DISCUSSION TITLE
API key

### DISCUSSION THREAD

**Discussion by hayleyzhou92** [lobehub/lobehub#3027] [General Question]
I added APIkey, but the model I created did not appear.

**Comment by arvinxx** [comment]
you need to fetch model list

**Comment by hayleyzhou92** [comment]
I fetch but nothing changed
---- Replied Message ----
FromArvin ***@***.***>Date6/25/2024 ***@***.***>***@***.***>,
***@***.***>SubjectRe: [lobehub/lobe-chat] API key (Discussion #3027)
you need to fetch model list

**Reply by arvinxx** [reply]
then you can add model by yourself

**Comment by hayleyzhou92** [comment]
how? I added the model here, but I can't use it. 
---- Replied Message ----
FromArvin ***@***.***>Date6/25/2024 ***@***.***>***@***.***>,
***@***.***>SubjectRe: [lobehub/lobe-chat] API key (Discussion #3027)
then you can add model by yourself
